pandas/tests/frame/test_reshape.py

```diff
@@ -936,3 +936,36 @@ def test_unstack_fill_frame_object():
         index=list('xyz')
     )
     assert_frame_equal(result, expected)
+
+
+def test_unstack_timezone_aware_values():
+    # GH 18338
+    df = pd.DataFrame({
+        'timestamp': [
+            pd.Timestamp('2017-08-27 01:00:00.709949+0000', tz='UTC')],
+        'a': ['a'],
+        'b': ['b'],
+        'c': ['c'],
+    })
```

Comment:
looks like this may need a `columns=['timestamp', 'a', 'b', 'c']` to ensure the right order on Py2.